### PR TITLE
[s] Adds fireworks to celebrate our new headmins

### PR DESCRIPTION
**Pull request by Supermichael777**
This is for April fools if you couldn't tell by grenades being as powerful as bombs were
:cl: Cubanpete
rcsadd: The Syndicate is celebrating the appointment of new heads to the secret counsel by improving the fireworks they ship to NT stations.
rcsadd: Spread the love with c4, grenades, and bombs.
rcsadd: C4's adhesive attaches to skin properly again.
/:cl:

### CODE REVIEW

File: code/game/machinery/syndicatebomb.dm [hunk 1]
```diff
@@ -190,7 +190,7 @@
 	if(adminlog)
 		message_admins(adminlog)
 		log_game(adminlog)
-	explosion(get_turf(src), 3, 9, 17, flame_range = 17)
```
Comment (ChangelingRain) [inline code/game/machinery/syndicatebomb.dm]: needs to ignore bombcap

**Comment by PKPenguin321** [comment]
>rcsadd: C4's adhesive attaches to skin properly again.

controversy inbound
(i'm hype though)

**Comment by Zilenan91** [comment]
Man no wonder they say that April Fools is the worst time to play this game

**Comment by Supermichael777** [comment]
Oh right the cap for pansys

**Comment by Core0verload** [comment]
Configs are NOT pulled from repo AFAIK. Any change to default config is pointless.

**Comment by Shadowlight213** [comment]
yeah. configs are not pulled

**Comment by duncathan** [comment]
@Supermichael777

**Comment by KorPhaeron** [comment]
Too late now

**Pull request closed without merging**